Express-API/department.js

```diff
@@ -1,4 +1,5 @@
 const getDepartmentsQuery = "SELECT name FROM department WHERE department_id = ?"
+const allDepartmentsQuery = "SELECT * FROM department"
```

Comment:
Never `SELECT *`. Only select the columns from the `department` table which you need